config/initializers/monkey_patches/chat.rb

```diff
@@ -0,0 +1,21 @@
+module Slack
```

Comment:
There is an issue with link [unfurling](https://api.slack.com/reference/messaging/link-unfurling#attaching_content) that requires a combination of `ts` and `channel`, or `unfurl_id` and `source`. The slack-ruby-client library has added validation to ensure that 'channel' and 'ts' are mandatory for link unfurling, as seen [here](https://github.com/slack-ruby/slack-ruby-client/blob/506376c8868561a9b2d936d12d6533da39985fdb/lib/slack/web/api/endpoints/chat.rb#L239): . However, we need to update the validation in the gem itself to make it work properly. The current version of the Ruby client is 2.0.0, but the latest version is 2.1.0. We will fork it, update it, and raise an upstream PR to make it live on the actual gem after the upgrade.